modules/virt-exposing-pci-device-in-cluster-cli.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-pci-passthrough.adoc
+
+[id="virt-exposing-pci-device-in-cluster-cli_{context}"]
+= Exposing PCI host devices in the cluster using the CLI
+
+To enable the availability of PCI host devices in the cluster, add details about the PCI devices to the `HyperConverged` custom resource (CR). The PCI devices are exposed and permitted to be used in the cluster.
+
+.Procedure
+. Edit the `HyperConverged` CR in YAML format using the default editor by running the following command:
++
+[source,terminal]
+----
+$ oc edit hyperconverged kubevirt-hyperconverged -n openshift-cnv -o yaml
+----
+
+. Add the PCI device information to the `spec.permittedHostDevices.pciHostDevices` array. For example:
++
+.Example configuration file
+[source,yaml]
+----
+apiVersion: hco.kubevirt.io/v1
+kind: HyperConverged
+metadata:
+  name: kubevirt-hyperconverged
+  namespace: openshift-cnv
+spec:
+  permittedHostDevices:
+    pciHostDevices:
+    - pciDeviceSelector: "8086:6F54" <1>
+      resourceName: "intel.com/qat" <2>
+      externalResourceProvider: true <3>
```

Comment:
Perhaps we shouldn't mention this here. In most cases, there is no need to use an external device plugin. 
Maybe we can just briefly mention that such an option exists.

`In case, the device requires a specialized device plugin, please add the externalResourceProvider: true  to the device configuration section` ?

Reply:
Vladik, yes it's rare, but we might want to keep it for completeness. Otherwise it will not pop-up anywhere in the docs.

Reply:
Agreed